shared/src/main/kotlin/org/jetbrains/kotlin/konan/target/ClangArgs.kt

```diff
@@ -155,7 +159,16 @@ class ClangArgs(private val configurables: Configurables) : Configurables by con
                         "-DKONAN_CORE_SYMBOLICATION=1",
                         "-DKONAN_HAS_CXX11_EXCEPTION_FUNCTIONS=1")
 
-            KonanTarget.IOS_ARM32, KonanTarget.IOS_ARM64 ->
+            KonanTarget.IOS_ARM32 ->
+                listOf("-DKONAN_OBJC_INTEROP=1",
+                        "-DKONAN_HAS_CXX11_EXCEPTION_FUNCTIONS=1",
+                        // While not 100% correct here, using atomic ops on iOS armv7 requires 8 byte alignment,
+                        // and general ABI requires 4-byte alignment on 64-bit long fields as mentioned in
+                        // https://developer.apple.com/library/archive/documentation/Xcode/Conceptual/iPhoneOSABIReference/Articles/ARMv6FunctionCallingConventions.html#//apple_ref/doc/uid/TP40009021-SW1
+                        // See https://github.com/ktorio/ktor/issues/941 for the context.
+                        "-DKONAN_NO_64BIT_ATOMIC=1")
+
+            KonanTarget.IOS_ARM64 ->
                 listOf("-DKONAN_OBJC_INTEROP=1",
                         "-DKONAN_HAS_CXX11_EXCEPTION_FUNCTIONS=1",
                         "-DKONAN_REPORT_BACKTRACE_TO_IOS_CRASH_LOG=1",
```

Comment:
Shouldn't these `KONAN_REPORT_BACKTRACE_TO_IOS_CRASH_LOG=1` and `DMACHSIZE` be also defined in `IOS_ARM32`?